SU-SWS/stanford_news
# Soeop 366

# READY FOR REVIEW

# Summary
- Adds banner background image, which is transparent, and update hook for install file for implementation?

# Needed By (Date)
- 11/2

# Urgency
- 3/10

# Steps to Test
1. clone https://github.com/SU-SOE/grunt-drupal-tasks and run `fin init`
2. Open a news page and notice the grey background.
3. Update `stanford.make` file located in `soe/src' folder. Change the following lines:
   i.  Replace `projects[stanford_news][download][tag] = "7.x-3.6+23-dev` with `projects[stanford_news][download][branch] = "SOEOP-366"`
   ii. Replace `projects[stanford_soe_helper][download][tag] = "7.x-2.1-alpha19` with `projects[stanford_soe_helper][download][branch] = "7.x-2.x-soeop-366"`
4. Run `git update` 
3. Open any news page or pages, and make sure the banner 's background is in accordance with the rest of the page. 


# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOEOP-366
- Left other similar function in for because [hook_update_N docs](https://api.drupal.org/api/drupal/modules%21system%21system.api.php/function/hook_update_N/7.x) recommend moving update hooks from two branches
- Assigning to @jbickar because I know his local setup, and how he can test. 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
